### ISSUE TITLE
Add travis CI

### ISSUE THREAD

**Issue by Aditya94A** [airmiha/create-shopify-app#3]
Adding travis CI would be pretty useful :)

**Comment by airmiha** [comment]
I have this in the pipeline. But, I'm leaning towards CircleCI. From my experience, it's easier to use. Also, people will be using this for commercial projects. CircleCI has a free plan so a CI setup would help people build their own pipelines. What do you think?

**Comment by Aditya94A** [comment]
Whichever one you prefer, both are good!

By the way, travis is also free for open source projects.

**Comment by airmiha** [comment]
Yeah, but when someone starts a private commercial project, they'll most likely choose CircleCI because it's free. In that case, they'll benefit from having a CI config file.
